evap/student/views.py

```diff
@@ -127,6 +126,10 @@ def vote(request, course_id):
                             answer_counter.add_vote()
                             answer_counter.save()
 
+        # enable text result publishing if first user in small course confirmed that
+        if request.POST.get('text_results_publish_confirmation') == 'on':
+            course.text_results_publish_confirmation = True
+            course.save()
```

Comment:
please add tests that let a single user vote with text answers (with and without checking the checkbox) and checks that e.g. a responsible can (not) see the text answers.